src/Microsoft.Health.Dicom.Core/Extensions/DicomDatasetExtensions.cs

```diff
@@ -62,13 +88,43 @@ public static T GetSingleValueOrDefault<T>(this DicomDataset dicomDataset, Dicom
         /// <param name="dicomDataset">The dataset to get the VR value from.</param>
         /// <param name="dicomTag">The DICOM tag.</param>
         /// <param name="expectedVR">Expected VR of the element.</param>
-        /// /// <remarks>If expectedVR is provided, and not match, will return null.</remarks>
-        /// <returns>An instance of <see cref="DateTime"/> if the value exists and comforms to the DA format; otherwise <c>null</c>.</returns>
-        public static DateTime? GetStringDateAsDate(this DicomDataset dicomDataset, DicomTag dicomTag, DicomVR expectedVR = null)
+        /// <remarks>If expectedVR is provided, and not match, will return null.</remarks>
+        /// <returns>An instance of <see cref="DateTimeOffset"/> if the value exists and comforms to the DA format; otherwise <c>null</c>.</returns>
+        public static DateTimeOffset? GetStringDateAsDate(this DicomDataset dicomDataset, DicomTag dicomTag, DicomVR expectedVR = null)
```

Comment:
Why change this to DateTimeOffset?

Reply:
This was done to make DT part of the range queries. DA was a DateTimeReaders supporting range queries. Since, the same could be used for DT, I ended up updating DA as offset.
For reference, please check ExtendedQueryTagDataRowsBuilder.cs.
As an alternative, a new reader can be created for DT avoiding changes for DA, but I felt like this way was cleaner. Hence, proceeded with it.

Reply:
I have changed everything back to DateTime. Now that I am using two separate requests for calculating local time and UTC time, DateTimeOffset is not required.